packages/authorization-process/src/services/authorizationService.ts

```diff
@@ -700,8 +703,11 @@ export function authorizationServiceBuilder(
       });
 
       return {
-        JWKKey: jwkKey,
-        client: client.data,
+        key: jwkKey,
+        client: clientToApiClient(client.data, {
+          includeKeys: false,
+          showUsers: false,
+        }),
```

Comment:
The downside of this approach is that the conversion happens here and not in the router. By the way, the `jwkKey` was already being converter here.